runtime/lua/vim/treesitter/dev.lua

```diff
@@ -115,11 +124,7 @@ function TSTreeView:new(bufnr, lang)
     ns = api.nvim_create_namespace('treesitter/dev-inspect'),
     nodes = nodes,
     named = named,
-    opts = {
-      anon = false,
-      lang = false,
-      indent = 2,
-    },
+    opts = default_treeview_opts,
```

Comment:
Do we need to split the default opts into a separate identifier? Now as someone reading the code, I have to jump to the `default_treeview_opts` definition, whereas before it was inline. Since this table is not re-used or referenced anywhere else, I think it's fine to leave it as-is, and I don't _think_ it should affect typing (if I'm wrong about that part though then that is a valid reason to move it to a separate identifier).

Reply:
After a bit of rebasing/rewriting this change is actually not needed but I thought it'd be convenient to have this closer to the type definition. It really doesn't matter much and would be a matter of code style, so let me revert this back.